release-notes/opensearch-documentation-release-notes-2.6.0.md

```diff
@@ -0,0 +1,19 @@
+# OpenSearch Documentation Website 2.6.0 Release Notes
+
+The OpenSearch 2.6.0 documentation includes the following additions and updates.
+
+## New documentation for 2.6.0
+
+- Add changes to remote backed storage documentation for 2.6 release [#3098](https://github.com/opensearch-project/documentation-website/pull/3098)
+- Add documentation for log type updates in Security Analytics - pt. 2 [#3083](https://github.com/opensearch-project/documentation-website/pull/3083)
+- Add documentation for log type updates in Security Analytics [#3066](https://github.com/opensearch-project/documentation-website/pull/3066)
```

Comment:
Not sure if it's really necessary. But could this one go above the "pt. 2" PR - 3083?